File: ios/MobilePassSDK/Delegate/BluetoothManagerDelegate.swift
//
//  BluetoothManagerDelegate.swift
//  MobilePassSDK
//
//  Created by Erinc Cakir on 9.02.2021.
//

public protocol BluetoothManagerDelegate {
    func onConnectionStateChanged(state: DeviceConnectionStatus)
    func onBLEStateChanged(state: DeviceCapability)
}

File: android/mobilepasssdk/src/main/java/com/armongate/mobilepasssdk/delegate/BluetoothManagerDelegate.java
package com.armongate.mobilepasssdk.delegate;

import com.armongate.mobilepasssdk.model.DeviceCapability;
import com.armongate.mobilepasssdk.model.DeviceConnectionStatus;

public interface BluetoothManagerDelegate {
    void onConnectionStateChanged(DeviceConnectionStatus state);
    void onBLEStateChanged(DeviceCapability state);
}
